### DISCUSSION TITLE
Does Jellyfin Media Player support SVP?

### DISCUSSION THREAD

**Discussion by justinluy** [jellyfin/jellyfin#5964] [Q&A]
I saw the post said "The software is also known to work with SVP with some tweaking of configuration files." But I have no idea how to do it. Can anyone please help? Thanks for your time.

**Comment by RosensRauk** [comment]
Did you get it to work?
I'm looking for a "how to"
/Fritz

**Comment by AetherVoyager** [comment]
any update regarding this ?

**Reply by nielsvanvelzen** [reply]
If there was an update it would've been posted here (although this is not the JMP repository).

_Sidenote: please change your username/profile picture to avoid confusion with the jellyfin organization_

**Reply by AetherVoyager** [reply]
Alright
